packages/graphql-toolbox/README.md

```diff
@@ -45,6 +45,20 @@ Run this command to start the development server. See the output in the console
 yarn start
 ```
 
+## GraphQL Toolbox URL parameters
+
+The GraphQL Toolbox supports some URL parameters defined in the query component, see below. These URL parameters are optional and can be provided at convenience.
+The query component is preceded by a question mark (?) and contains a query string that is a sequence of key-value pairs separated by an ampersand (&).
+
+| URL parameter | Description                                                                                                                             | Example                     |
+| ------------- | --------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- |
+| connectURL    | Form: scheme://username@bolt-url:port; Used at login, pre-populates the Username and Connection URI input field with the provided value | bolt://admin@localhost:7687 |
+| db            | Only applicable for multi-database supported Neo4j DBMSs. The provided value is used as the selected database name                      | neo4j                       |
+
+An example with the URL parameters `connectURL` and `db`: `http://localhost:4242?connectURL=bolt%2Bs://xxxx.databases.neo4j.io&db=mydatabase`
+
+Note: The plus symbol (+) in the connectURL (here in `bolt+s`) needs to be URL encoded with `%2B`.
```

Comment:
```suggestion
Note: The plus symbol (+) in the connectURL needs to be URL encoded to become `%2B`. For example `bolt+s` becomes `bolt%2Bs`.
```
I think this may improve clarity.